sdk/template/template/package.json

```diff
@@ -87,7 +87,7 @@
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^2.0.1",
-    "@azure-tools/test-recorder": "^1.0.0",
```

Comment:
was the dependence on both the old and the new at the same time just a mistake or does the template package actually use both?

Reply:
Wasn't a mistake when the tests were migrated.

Now the recorder 2.0 is self-sufficient, doesn't need pieces from the old recorder anymore.